src/main/java/uk/gov/justice/digital/client/glue/GlueClient.java

```diff
@@ -10,12 +10,10 @@
 import org.jetbrains.annotations.NotNull;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.datahub.model.SourceReference;
 import uk.gov.justice.digital.exception.GlueClientException;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
prefer explicit imports rather than blanket (but that is just me).

Reply:
This is now fixed. Thanks